GoogleForCreators/web-stories-wp
# Workspace UX: Move Page Menu

## Summary
- Moves the history buttons to the header
- Moves the other page menu buttons to the side of the page
- Restyles the full side menu in Zoom mode

Note: removes the Page number fully currently as this will need reworking in #9641 anyway.

Adjusting the page size to use the new available space will be done in #9642 


## Relevant Technical Choices
- Creates a new `PageSideMenu` component that includes both the quick actions menu and the Page buttons.


## To-do




## User-facing changes
| State | Before | After |
| ----- | -------| ----- |
| Page: Default | <img width="376" alt="Screenshot 2021-11-11 at 15 39 26" src="https://user-images.githubusercontent.com/3294597/141307849-2acb1e0a-4c50-419d-a090-3e4ec27d9044.png"> | <img width="431" alt="Screenshot 2021-11-11 at 15 50 29" src="https://user-images.githubusercontent.com/3294597/141309413-caf15764-3606-4b1b-96ce-d92396248b5c.png"> | 
| Zoom 100% | <img width="736" alt="Screenshot 2021-11-11 at 15 39 40" src="https://user-images.githubusercontent.com/3294597/141307844-73362f9a-5e68-42a8-83cb-708f795be89b.png"> | <img width="728" alt="Screenshot 2021-11-11 at 15 36 12" src="https://user-images.githubusercontent.com/3294597/141307462-ba68b9eb-c693-4685-8a11-01f93c3b15b9.png"> |
| Zoom 200% | <img width="738" alt="Screenshot 2021-11-11 at 15 39 52" src="https://user-images.githubusercontent.com/3294597/141307835-fd819b75-7199-4593-b227-ccebc6e7a6aa.png"> | <img width="738" alt="Screenshot 2021-11-11 at 15 36 28" src="https://user-images.githubusercontent.com/3294597/141307454-1473a44f-8482-4ec9-929f-145a4f7caa29.png"> |



## Testing Instructions





This PR can be tested by following these steps:
See [Figma](https://www.figma.com/file/dc9xrVlRhQq4KTE9WQzU1R/_Editor---Sprint-Feature-Level-Designs_?node-id=10548%3A453241) for style.

1. Verify that Undo and Redo buttons are now in the header as per Figma. Verify the buttons still work
2. Without Zoom: Verify that the Page Menu buttons are now on the right bottom side of the Page
3. With Zoom: Verify that the Quick Actions menu and the Page Menu buttons are always visible (no matter the Zoom %) and are on the right side of the Workspace.

## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #9640
